client/common/sass/components/_search-bar.sass

```diff
@@ -0,0 +1,9 @@
+.search-form
+	position: relative
+
+.search-button
+	position: absolute
+	right: 4px
+	top: 5px
```

Comment:
I think `top: 3.5px` and `right: 2px` (since border is 2px) looks more close to the design.

Reply:
Also, add:

```sass
&:focus
  top: 1px
```